drivers/inactive_client_report/app/controllers/inactive_client_report/warehouse_reports/reports_controller.rb

```diff
@@ -25,7 +27,11 @@ class ReportsController < ApplicationController
     def index
       @excel_export = ::InactiveClientReport::DocumentExports::ReportExcelExport.new
       respond_to do |format|
-        format.html {}
+        format.html do
+          # This is being set so the render_inline feature can be used in development
+          # to override the background running of the report
+          set_pagy_data if params[:render_inline] == '1' && Rails.env.development?
```

Comment:
In this case, maybe we should remove the development restriction so we can see performance in production?  At least temporarily?